lib/openhab/core.rb

```diff
@@ -21,6 +21,21 @@ def self.version
       @version ||= Gem::Version.new(VERSION).release.freeze
     end
 
+    #
+    # Returns the raw version of openHAB
+    #
+    # The {version} method returns the release version, stripping off any
+    # additional qualifiers such as M1, or snapshots.
+    # This method returns the full version string, including the qualifiers.
+    #
+    # @return [Gem::Version] Returns the raw version of openHAB
+    #
+    # @!visibility private
+    #
+    def self.raw_version
```

Comment:
let's call this `full_version`, not `raw_version`

Reply:
changed